variables.tf

```diff
@@ -100,3 +107,10 @@ variable "zone" {
   description = "The primary zone where the bastion host will live"
   default     = "us-central1-a"
 }
+
+variable "random_role_id" {
+  description = "Enables role random id generation."
+  type        = bool
+  default     = false
```

Comment:
I think we could default to `true`? Given the difficulties with recreating custom roles, this seems like a safer default.

Reply:
done